src/test/ui/macros/issue-68058.rs

```diff
@@ -0,0 +1,15 @@
+// check-pass
+
+macro_rules! def_target {
+    ($target: expr) => {
+        #[target_feature(enable=$target)]
```

Comment:
This attribute is not relevant to the issue and can be removed.

Reply:
`unsafe` on the function is irrelevant as well.

Reply:
Removed.